spec/uncompiled/app_multiple.rabl

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+extends 'extend'
+
+extends 'extend'
```

Comment:
I would remove this dup.

Reply:
It is specifically testing having multiple extends.